examples/image_classifier/squeezenet/README.md

```diff
@@ -0,0 +1,48 @@
+#### Sample commands to create a squeezenet eager mode model archive, register it on TorchServe and run image prediction
+
+Run the commands given in following steps from the parent directory of the root of the repository. For example, if you cloned the repository into /home/my_path/serve, run the steps from /home/my_path
```

Comment:
This instruction doesn't look correct. Example below should be `--model-file serve/examples/image_classifier/squeezenet/model.py` instead of `--model-file examples/image_classifier/squeezenet/model.py`

Reply:
Fixed the instructions.